.github/workflows/integration-testing.yaml

```diff
@@ -31,3 +31,4 @@ jobs:
           AZURE_CLIENT_ID: ${{ secrets.SBX_CLIENT_ID }}
           ARM_CLIENT_SECRET: ${{ secrets.SBX_CLIENT_SECRET }}
```

Comment:
I never understood the difference between ARM_ and AZURE_

Reply:
I know it's just a naming convention, but do we use both?

Reply:
Yes.  One is for the `azurerm` provider in the Terraform fixture (the `ARM_` prefix), and the other is for the azure Go module (the `AZURE_` prefix).  They are basically the same thing - credentials for the Azure Service Principal identity backing the automation pipeline.